blocks/footer/footer.css

```diff
@@ -1,6 +1,7 @@
 footer {
   padding: 2rem;
-  font-size: var(--mt-body-font-size-s);
+  background-color: var(--mt-background-color-primary);
+  font-size: var(--mt-font-size-100);
```

Comment:
please use `--mt-body-font-size-m` - globals should only be referenced implicitly.